json-rpc/eth_getBalance.md

```diff
@@ -0,0 +1,17 @@
+# `Eth_getBalance`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1|Eth_getBalance must be sent a valid Ethereum address as the first parameter|
+1.1 |Eth_getBalance should return error code -32602 "invalid argument 0: json: cannot unmarshal hex string of odd length into Go value of type common.Address" if the address send has an odd length|
```

Comment:
interestingly this message looks very bad and seems to be an internal Geth error so a better message should be suggested